pir/pircl/doc_test.go

```diff
@@ -0,0 +1 @@
+package pircl
```

Comment:
With our build flags, the OpenCL and CUDA implementations are ignored for build/test on Travis. However, tests and goveralls will still look for a test report from these folders (which appear empty). See
https://travis-ci.org/privacylab/talek/jobs/230595872

I added doc.go and doc_test.go to keep the tests happy.